### PR TITLE
use-case-1-odu2-service-01.json: fixed comment breakup

### PR DESCRIPTION
**Pull request by GianmarcoBruno**

### CODE REVIEW

**Comment by GianmarcoBruno** [comment]
Updated the json for the use-case using array notation, i.e.
"// comment-string " : ["string1", "string2", "string3"]

**Comment by GianmarcoBruno** [comment]
Updated the json for the use-case using array notation, i.e.
"// comment-string " : ["string1", "string2", "string3"]

**Comment by italobusi** [comment]
@GianmarcoBruno : I agree.
Could you please fix the code together with the other comments?
Thanks

File: Internet-Drafts/Use-Case-1-Analysis/01/use-case-1-topology-01.json [hunk 1]
```diff
@@ -1,218 +1,1173 @@
...
+                "ietf-te-topology:te": {
+                  "state": {
                     "// adaptation-type": " ccamp-otn-topo-yang-00",
+                    "// comment": [
+                     "Agumentation from",
+                      " draft-ietf-ccamp-otn-topo-yang-00"
+                    ],
                     "// sink-adapt-active": "To be discussed",
                     "// source-adapt-active": "To be discussed",
-                    "// tributary-slots": "To be added (80 available",
```
[the remark this answers is not in the document]
Comment (italobusi) [reply Internet-Drafts/Use-Case-1-Analysis/01/use-case-1-topology-01.json]: It seems that the comment was not aligned in old JSON code
It is proposed to have the following comments in all the LTPs:
`"// tributary-slots": " TSs at 1,25TS granularity (80 available)",`

File: Internet-Drafts/Use-Case-1-Analysis/01/use-case-1-topology-01.json [hunk 2]
```diff
@@ -1,218 +1,1173 @@
...
-          }
-        },
         "ietf-te-topology:te": {
           "state": {
             "// comment": "None analysed up to so far"
           }
         },
         "node": [
           {
-            "// comment": "S3 Node",
```
[the remark this answers is not in the document]
Comment (GianmarcoBruno) [reply Internet-Drafts/Use-Case-1-Analysis/01/use-case-1-topology-01.json]: OK I just replaced all the original "node : [..]" and "link : [..]" structures from those by Ricard Vilalta. So these comments got lost. I think that once we have the topology in place and the order of its fields is preserved, we should be able to keep them from one revision to another.

**Pull request merged**